app/views/govuk_publishing_components/components/docs/govspeak.yml

```diff
@@ -911,3 +912,28 @@ examples:
             <p>Deforested area. Credit: Blue Ventures-Garth Cripps</p>
           </figcaption>
         </figure>
+  different_font_sizes:
+    description: |
+      Set a different font size for the govspeak. The only valid option is `l`.
+      
+      This option should be used rarely, in cases where the design calls for a short
+      piece of large rich text (for example, in the content of a hero block).
+    data:
+      block: |
+        <p>This is large govspeak</p>
+        <p>It is:</p>
+        <ul>
+          <li>Bigger than normal govspeak</li>
+          <li>Intended to be used rarely</li>
+          <li>A way to include large rich text when the design calls for it</li>
+        </ul>
+        <div class="call-to-action">
+          <p>It should support all the usual govspeak features, such as calls to action.</p>
+        </div>
+        <h1>Heading 1</h1>
```

Comment:
I think you can remove `h1` as I don't think we do H1s in page content.

Reply:
Addressed in [91389d3](https://github.com/alphagov/govuk_publishing_components/pull/4461/commits/91389d3508e8f3a404ecc9fc8ee13178c1daa17e)